### ISSUE TITLE
Feature Request: Option to download mp3

### ISSUE THREAD

**Issue by frakman1** [rroller/media-roller#4]
Adding the `-x --audio-format mp3` option to youtube-dl downloads the MP3 file. Is there anyway you can integrate that feature please?

**Comment by d8sychain** [comment]
It should be doable. I bashed into the container and used the command: 
` docker exec -it media-roller /bin/bash`
` youtube-dl -x --audio-format mp3 -o "/download/%(title)s.%(ext)s" URL`
You can also:
` docker exec -it media-roller youtube-dl -x --audio-format mp3 -o "/download/%(title)s.%(ext)s" URL`

Add an option button or a check box on the WebUI to add `-x --audio-format mp3` to the command string passed to youtube-dl.

**Comment by frakman1** [comment]
I switched to [this](https://github.com/Tzahi12345/YoutubeDL-Material) project instead. It has this exact feature.

![image](https://user-images.githubusercontent.com/5826484/118650458-7d595400-b7b2-11eb-9647-ffeb30c7bcbd.png)

It also features multi-user support, subscriptions etc.

**Comment by rroller** [comment]
Yes, youtubedl-material is more appropriate for your use case. This project (media-roller) is pretty much geared to be an API I can call from my iPhone to 1 tap download any video so I didn't put any effort into the UI. The UI was mainly around testing.

**Comment by frakman1** [comment]
That's really cool. How are you using it with you iPhone? I'd like to use something like that. Mine is jailbroken.

**Comment by rroller** [comment]
@frakman1 I've documented it in the readme... https://github.com/rroller/media-roller/blob/master/README.md

Basically I run this service on my home network (and use a VPN to access it when I'm away)... I then used this shortcut to download videos for URLs that I have on my clipboard... https://www.icloud.com/shortcuts/2b9765fb5bf24509a4e83bc76c5c7409

So for example, I'll copy a YouTube URL and click the shortcut and it download the video to my camera roll.

**Comment by frakman1** [comment]
That's so cool! I love that you used Shortcuts and Youtube and bent them to your will.haha
Well done!
